src/PreferencesDialog.cpp

```diff
@@ -561,3 +564,13 @@ void PreferencesDialog::updatePreviewFont()
         ui->txtBlob->setFont(textFont);
     }
 }
+
+void PreferencesDialog::on_buttonManageFileExtension_clicked()
```

Comment:
If the user edits the extensions but changes his mind and enters again in the dialog, the last edited extensions are not displayed, but the initial set. The new dialog should be populated with m_dbFileExtensions and not with the saved setting.

Reply:
Ok